graftm/pplacer.py

```diff
@@ -50,16 +50,23 @@ def jplace_split(self, jplace_file, alias_hash):
 
         # Load the placement file
         placement_file = json.load(open(jplace_file))
-
+        
         # Parse the placements based on unique identifies appended to the end
         # of each read
         for placement in placement_file['placements']: # for each placement
             hash = {} # create an empty hash
```

Comment:
this statement does nothing because `hash` is always re-instantiated later anyway